src/service/sanity.ts

```diff
@@ -3,7 +3,7 @@ import SanityClient from '@sanity/client';
 export const sanityClient = SanityClient({
   // Find your project ID and dataset in `sanity.json` in your studio project
   projectId: 'jgfkj4nx',
-  dataset: 'production'
```

Comment:
I dont think this is nessecsary change :)

Reply:
Before I set it to certificates-dev, none of the main content of the certificates page would show up for me.  It was just the header and the footer.

Reply:
Ah kk, I will make sure the right dataset is on production :)

Reply:
It is now updated so you can go back to production :)